engine/lite/kafka/runtime/src/main/scala/pl/touk/nussknacker/engine/lite/kafka/NuKafkaRuntimeApp.scala

```diff
@@ -66,12 +67,19 @@ object NuKafkaRuntimeApp extends App with LazyLogging {
 
     val modelData = ModelData(modelConfig, ModelClassLoader(modelConfig.as[List[URL]]("classPath")))
 
-    val metricRegistry = LiteEngineMetrics.prepareRegistry(engineConfig)
+    val metricRegistry = prepareMetricRegistry(engineConfig)
     val preparer = new LiteEngineRuntimeContextPreparer(new DropwizardMetricsProviderFactory(metricRegistry))
     // TODO Pass correct ProcessVersion and DeploymentData
     val jobData = JobData(scenario.metaData, ProcessVersion.empty, DeploymentData.empty)
 
     KafkaTransactionalScenarioInterpreter(scenario, jobData, modelData, preparer)
   }
 
+  private def prepareMetricRegistry(engineConfig: Config) = {
+    lazy val instanceId = sys.env.get("INSTANCE_ID") orElse sys.env.get("HOSTNAME") getOrElse {
+      throw new IllegalStateException("Cannot determine instanceId: neither INSTANCE_ID nor HOSTNAME environment variables unavailable")
```

Comment:
can we do here some generic fallback - I wonder if throwing exception would be to harsh ?

Reply:
I've changed it a bit - now it is fallback to "localhost".